awkward-cpp/include/awkward/common.h

```diff
@@ -30,9 +30,8 @@
 
 #define QUOTE(x) #x
 
-#define FILENAME_FOR_EXCEPTIONS(filename, line) std::string("\n\n(https://github.com/scikit-hep/awkward-1.0/blob/" VERSION_INFO "/" filename "#L" #line ")")
-#define FILENAME_FOR_EXCEPTIONS_C(filename, line) ("\n\n(https://github.com/scikit-hep/awkward-1.0/blob/" VERSION_INFO "/" filename "#L" #line ")")
-#define FILENAME_FOR_EXCEPTIONS_CUDA(filename, line) ("\n\n(https://github.com/scikit-hep/awkward-1.0/blob/" QUOTE(VERSION_INFO) "/" filename "#L" #line ")")
+#define FILENAME_FOR_EXCEPTIONS_C(filename, line) "\n\n(https://github.com/scikit-hep/awkward-1.0/blob/awkward-cpp-" VERSION_INFO "/awkward-cpp/" filename "#L" #line ")"
```

Comment:
We haven't done a general update of URLs within the codebase, but while we're touching this one, we might as well fix it. (This one, in particular, is visible to users.)

```suggestion
#define FILENAME_FOR_EXCEPTIONS_C(filename, line) "\n\n(https://github.com/scikit-hep/awkward/blob/awkward-cpp-" VERSION_INFO "/awkward-cpp/" filename "#L" #line ")"
```

Reply:
oop, yep!